### DISCUSSION TITLE
Any plans on adding Oracle linux ?

### DISCUSSION THREAD

**Discussion by virtualistic** [vmware/packer-examples-for-vsphere#659] [Ideas]
Was wondering if there are any plans on adding Oracle Linux versions to the packer examples?

**Comment by tenthirtyam** [comment]
None planned, but contributions are welcome!

**Reply by virtualistic** [reply]
I created a new branch on my forked project. I'll create a pull request.
Never done this on a public project so please bear with me :)
Also sharing some handy links that one might need when creating a new template that are not easy to find:
https://discuss.hashicorp.com/t/list-of-valid-guest-os-type-values-for-vsphere-builder/4671/14
https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_conditionals.html

**Reply by tenthirtyam** [reply]
Thank for the contribution! 🎉 

- https://github.com/vmware-samples/packer-examples-for-vsphere/pull/670
